.bazelci/presubmit.yml

```diff
@@ -24,3 +24,14 @@ tasks:
     test_flags:
       - "--config=coverage"
       - "--config=clang"
+  fuzz_coverage:
+    name: "Fuzz-Coverage"
+    platform: ubuntu1804
+    shell_commands:
+      - "bazel/setup_clang.sh /usr/lib/llvm-10"
+    test_targets:
+      - "//test/server:server_fuzz_test"
```

Comment:
Do resources matter for bazel CI?
On one hand this is a good target since it builds the most, on the other hand, if timeouts or resources are a concern, this tends to be a very, very slow build.

Reply:
AFAIK it runs on a powerful machine (32 cores AFAIK) and heavily cached with RBE, so I think this will fine, their CI runs this at daily and for release candidates.

This is to catch issues like https://github.com/bazelbuild/bazel/issues/12545 before Bazel release.